Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.11.6-alpine3.18
+
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR app/
+
+COPY . .
+RUN pip install -r requirements.txt
+
+CMD ["python", "app/main.py"]
```

Comment:
The Dockerfile does not follow best practices
https://docs.docker.com/build/building/best-practices/